test/TodoTest.java

```diff
@@ -0,0 +1,10 @@
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+class TodoTest {
+    @Test
+    void testToString() {
+    Todo newTodo = new Todo("Read Asura Omega manga");
+    assertEquals("[T][✘] Read Asura Omega manga",newTodo.toString());
```

Comment:
I think there should have 1 space between `...Omega manga",` and `newTodo...`

Reply:
I have gone through all of your comments and I found then really helpful. Thank you for making the effort to go through all the fine details, I appreciate it 👍

Reply:
@aidoxe-123, I will also make the relevant changes for the code quality guidelines you have pointed out to me as soon as possible in the next commit!